.travis.yml

```diff
@@ -5,6 +5,11 @@ sudo: false
 cache:
   directories:
     - $HOME/.composer/cache
+    - $HOME/.npm
+
+env:
+  global:
+    - NPM_CONFIG_PROGRESS='false'
```

Comment:
It doesn't seem to have an effect in Travis CI environment anyway, since there's no TTY :stuck_out_tongue:

But there's no harm...